biz.aQute.resolve/src/biz/aQute/resolve/BndrunResolveContext.java

```diff
@@ -379,6 +379,19 @@ private Processor loadRepositories() throws Exception {
 		augments.putAll(new Parameters(properties.mergeProperties(Constants.AUGMENT), project));
 
 		if (!augments.isEmpty()) {
+
+			if (project != null) {
+				String distro = properties.mergeProperties(Constants.DISTRO);
```

Comment:
Here we are grabbing the distro again.  Should I try to share the code elsewhere when the distro bundle is looked up?